pyiron_workflow/node_library/dev_tools.py

```diff
@@ -0,0 +1,87 @@
+# for development and testing only
+# provide functionality, data types etc. that will be later moved to the workflow code
+
+from pathlib import Path
+
+
+class VarType:
+    def __init__(
+        self,
+        value=None,
+        dat_type=None,
+        label: str = None,
+        store: int = 0,
+        generic: bool = None,
+        doc: str = None,
+    ):
+        self.value = value
+        self.type = dat_type
+        self.label = label
+        self.store = store
+        self.generic = generic
+        self.doc = doc
+
+
+class FileObject:
```

Comment:
This is nearly the same object as the [`FileObject`](https://github.com/pyiron/pyiron_workflow/blob/bb79722dcc0d0263f5514fa52c0cd767e501bcf4/pyiron_workflow/files.py#L75) that we already have in [`files.py`](https://github.com/pyiron/pyiron_workflow/blob/main/pyiron_workflow/files.py)